test/Conversion/ExportVerilog/name-legalize.mlir

```diff
@@ -11,9 +11,9 @@ hw.module @namechange(%casex: i4) -> (if: i4) {
 hw.module.extern @module_with_bool<bparam: i1>() -> ()
 
 // CHECK-LABEL: module parametersNameConflict
-// CHECK-NEXT:    #(parameter [41:0] p1_0 = 42'd17,
+// CHECK-NEXT:    #(parameter [41:0] p1 = 42'd17,
 // CHECK-NEXT:      parameter [0:0]  wire_0) (
-// CHECK-NEXT:    input [7:0] p1);
+// CHECK-NEXT:    input [7:0] p1_0);
 hw.module @parametersNameConflict<p1: i42 = 17, wire: i1>(%p1: i8) {
   %myWire = sv.wire : !hw.inout<i1>
```

Comment:
Can you instead rewrite the test to not have a conflict? We may want to error/warn on port/param collisions in the future.